public.redis.tf

```
@@ -0,0 +1,61 @@
+# Public Redis Instance
+resource "azurerm_resource_group" "public_redis" {
+  name     = "public-redis"
+  provider = azurerm.jenkins-sponsorship
+  location = var.location
+  tags     = local.default_tags
+}
+
+# Redis database
+resource "azurerm_redis_cache" "public_redis" {
+  name                          = "public-redis"
```

```suggestion
  name                          = "public-redis"
  provider = azurerm.jenkins-sponsorship
```
=> need to use the sponsored subscription

(dont forget to `terraform fmt -recursive after this suggestion)